templates/minio/minio.py

```diff
@@ -66,7 +66,6 @@ def node_port(self):
 
     def install(self):
         self.logger.info('Installing minio %s' % self.name)
-        minio_sal = self._minio_sal
```

Comment:
why is this removed ?

Reply:
Because it doesn't do anything. Doesn't even create the container.